presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcSplit.java

```diff
@@ -28,12 +29,15 @@
         implements ConnectorSplit
 {
     private final Optional<String> additionalPredicate;
+    private final Optional<RangeInfo> rangeInfo;
 
     @JsonCreator
     public JdbcSplit(
-            @JsonProperty("additionalPredicate") Optional<String> additionalPredicate)
+            @JsonProperty("additionalPredicate") Optional<String> additionalPredicate,
+            @JsonProperty("rangeInfo") Optional<RangeInfo> rangeInfo)
```

Comment:
Can we use `additionalPredicate` instead?
I am aware `rangeInfo` allows you to bind the range ends in prepared statement, but as long as you support numbers only, this doesn't add much benefit.